tests/runtime/engine/runner.py

```diff
@@ -321,11 +335,26 @@ def run_test(test):
                 print('\tCLEANUP error: %s' % e.stderr)
                 return Runner.FAIL
 
+        def print_befores_and_after_output():
+            if len(befores_output) > 0:
+                print("\tBefore cmd output:")
+                for (out, err) in befores_output:
+                    out = out.decode("utf-8").replace('\n', '\n\t\t\t')
+                    err = err.decode("utf-8").replace('\n', '\n\t\t\t')
```

Comment:
I wouldn't do this. Sometimes, it's useful to see the exact output of the commands and adding artificial whitespace doesn't help much.

Reply:
+1. I think we should follow the example below:

```
print('\tFound: ' + output.encode("unicode_escape").decode("utf-8"))
```

That is, encode all the whitespace as escaped characters so it doesn't scribble all over the screen. Makes output tidier.

Reply:
Changed to use the unicode escape